src/scene_logo.cpp

```diff
@@ -120,11 +120,14 @@ void Scene_Logo::OnIndexReady(FileRequestResult*) {
 	tree->SetImportantFile(true);
 	FileRequestAsync* ini = AsyncHandler::RequestFile(INI_NAME);
 	ini->SetImportantFile(true);
+	FileRequestAsync* exe = AsyncHandler::RequestFile(EXE_NAME);
+	exe->SetImportantFile(true);
```

Comment:
This should be reordered I think.

Reply:
can be removed, as emscripten won't parse RPG_RT.exe and exfont loading is already below